coreplugins/cesiumion/public/components/BootstrapField.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const BootstrapFieldComponent = ({
+    name,
+    value,
+    label,
+    help,
+    type = "text",
+    showIcon = true,
+    error,
+    touched,
+    onChange,
+    onBlur,
+    ...props
+}) => {
+    const isError = error && touched;
+    const isCheckbox = type === "checkbox";
+    const ControlComponent = isCheckbox ? "input" : (type === "textarea" || type === "select") ? type : "input";
+
+    return (
+        <div className={`form-group${isError ? ' has-error' : ''}`} style={{ marginLeft: 0, marginRight: 0 }}>
+            {label && !isCheckbox && <label htmlFor={name} className="control-label">{label}</label>}
```

Comment:
Can we use a conditional to return a new component for the checkbox instead of using the `isCheckbox` variable multiple times?